deploy/nebula/templates/clusterdefinition.yaml

```diff
@@ -0,0 +1,199 @@
+apiVersion: apps.kubeblocks.io/v1alpha1
+kind: ClusterDefinition
+metadata:
+  name: nebula
+  labels:
+    {{- include "nebula.labels" . | nindent 4 }}
+spec:
+  connectionCredential:
+    username: root
+    password: "$(RANDOM_PASSWD)"
+    endpoint: "$(SVC_FQDN):$(SVC_PORT_tcp-nebula)"
+    host: "$(SVC_FQDN)"
+    port: "$(SVC_PORT_tcp-nebula)"
+    accesskey: ""
+    secretkey: ""
+  componentDefs:
+    - name: nebula-metad
+      configSpecs:
+        - name: nebula-metad-config
+          templateRef: nebula-metad-config-template
+          volumeName: nebula-metad
+      scriptSpecs:
+        - name: nebula-metad-scripts
+          templateRef: nebula-metad-scripts-template
+          volumeName: scripts
+          defaultMode: 493
+      workloadType: Stateful
+      characterType: nebula-metad
+      podSpec:
+        containers:
+          - name: nebula-metad
+            command:
+              - /bin/bash
+              - -c
+              - |
+                set -ex
+                /scripts/nebula-metad-start.sh
+            imagePullPolicy: {{default .Values.nebula.metad.image.pullPolicy "IfNotPresent"}}
+            ports:
+            - containerPort: 9559
+              name: thrift
+              protocol: TCP
+            - containerPort: 19559
+              name: http
+              protocol: TCP
+            - containerPort: 19560
+              name: http2
+              protocol: TCP
+            readinessProbe:
+              failureThreshold: 3
+              httpGet:
+                path: /status
+                port: http
+                scheme: HTTP
+              initialDelaySeconds: 10
+              periodSeconds: 10
+              successThreshold: 1
+              timeoutSeconds: 5
+            volumeMounts:
+              - name: scripts
+                mountPath: /scripts
+              - mountPath: /usr/local/nebula/data
+                name: data
+              - mountPath: /usr/local/nebula/logs
+                name: logs
+              - mountPath: /usr/local/nebula/etc
+                name: nebula-metad
+        volumes:
+          - emptyDir: {}
```

Comment:
there is no need to explicitly declare emptyDir volumes, KubeBlocks cluster operator will automatically inject emptyDir volume if volumeMounts' name has no associate volume.

Reply:
Fixed.